### DISCUSSION TITLE
Perkenalan dengan Grav CMS - Topidesta - Tulisan Programmer Gadungan

### DISCUSSION THREAD

**Discussion by giscus** [topidesta/topidesta#4] [Comments]
# Perkenalan dengan Grav CMS - Topidesta - Tulisan Programmer Gadungan

Bismillah .... Kebetulan lagi pengen cari Alternatif FlatFile CMS, hampir semuanya mengarah ke MERN Stack. Kebetulan PHP masih menjadi…

https://topidesta.my.id/perkenalan-dengan-grav-cms

**Comment by mdestafadilah** [comment]
Ouh ouh ... lanjut lagi pelajarin... enak juga...

**Reply by muhammad-rifqi** [reply]
masa ???

**Reply by mdestafadilah** [reply]
iya da, cek aja websitenya. lengap dokumentasinya.

**Reply by mdestafadilah** [reply]
uda nulis lagi dong..

**Reply by muhammad-rifqi** [reply]
tentu des

**Reply by mdestafadilah** [reply]
mana gak apdet apdet lu

**Reply by muhammad-rifqi** [reply]
masalahnya gw ng tau dimana nulisnya .. wkwkwk

**Reply by mdestafadilah** [reply]
akhirnya komentar bisa lagi

**Reply by mdestafadilah** [reply]
uda @muhammad-rifqi saur saur saur

**Comment by mdestafadilah** [comment]
kalo mau liat hasil pengembangan gravcms

https://v3.web.persahabatan.co.id/id

**Comment by mdestafadilah** [comment]
sekarang website halobayi yang pake gravcms guys
